build_tarballs.jl

```diff
@@ -45,9 +47,10 @@ platforms = [
 ]
 
 # The products that we will ensure are always built
-products(prefix) = [
-    ExecutableProduct(prefix, "qgen-delve", :qgen_delve),
-    ExecutableProduct(prefix, "dbgen-delve", :dbgen_delve)
+products = [
+    ExecutableProduct("qgen-rel", :qgen_rel, "tpch-dbgen"),
+    ExecutableProduct("dbgen-rel", :dbgen_rel, "tpch-dbgen"),
+    ExecutableProduct("qgen-delve", :qgen_delve, "tpch-dbgen"),
```

Comment:
Won't `dbgen-delve` also be needed here?

Reply:
Nope, as the code is the same for dbgen. So in the PR for RAI, we apply that renaming already.